DigitalLearningSolutions.Data/Models/User/UserCentreAccounts.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace DigitalLearningSolutions.Data.Models.User
+{
+    public class UserCentreAccounts
+    {
+        public UserCentreAccounts(
+            int centreId,
+            AdminAccount? adminAccount = null,
+            DelegateAccount? delegateAccount = null
+        )
+        {
+            CentreId = centreId;
+            AdminAccount = adminAccount;
+            DelegateAccount = delegateAccount;
+        }
+
+        public int CentreId { get; }
+        public AdminAccount? AdminAccount { get; }
+        public DelegateAccount? DelegateAccount { get; }
+        public bool IsCentreActive => AdminAccount?.CentreActive ?? DelegateAccount?.CentreActive ?? false;
+
+        public bool IsActive => (AdminAccount != null || DelegateAccount != null) &&

```

Comment:
I have removed this and replaced it with `CanLogDirectlyInToCentre`, which covers the use case in a (I think) less confusing way. 

I've also added `CanLogInToCentre`, the distinction being that `CanLogDirectlyInToCentre` will allow you to go from login -> a single centre is automatically selected, in accordance with 898:
> * When a user successfully attempts to log into a User account with only accounts at a single centre or by logging in with a CandidateNumber:
>   * If the centre they are trying to log into is inactive, do a basic login and redirect to the Choose a Centre page.
>   * If either the Admin or Delegate account at the centre is inactive, do a basic login and redirect to the Choose a Centre page.
>   * If the delegate account (if it exists) is unapproved, do a basic login and redirect to the Choose a Centre page.
>   * Otherwise log directly into that centre.

while `CanLogInToCentre` is to check whether you will get a Choose button on the choose a centre page, and whether the ChooseCentre route will work for you

Reply:
I've updated the ticket. Swiki is correct: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/19467173889/Sign+in#Possible-page-outcomes.1

Reply:
I have now removed `CanLogDirectlyInToCentre`, since `CanLogInToCentre` provides the expected behaviour ("If all the user's accounts at the centre are inactive...")